### PR TITLE
Update EPUB parsing doc

### PR DESCRIPTION
**Pull request by JayPanoz**
Presentation hints + spine overrides. 

Please let me know if something is wrong/obscure.

### CODE REVIEW

File: streamer/parser/metadata.md [hunk 1]
```diff
@@ -291,48 +293,68 @@ Does not apply.
 
 The integer is the value of the `meta` element whose `property` attribute has the value `schema:numberOfPages`.
 
-## Rendition
+## Rendition / Presentation
 
-The `rendition` of a publication is an object containing the following keys: 
+The `presentation` of a publication is an object containing the following keys:
```
Comment (HadrienGardeur) [inline streamer/parser/metadata.md]: We should also list `layout`.

Comment (JayPanoz) [reply streamer/parser/metadata.md]: Ah if you can have it in presentation (wasn't sure) then it also simplify the layout section.

Comment (HadrienGardeur) [reply streamer/parser/metadata.md]: Yes you can: https://readium.org/webpub-manifest/schema/extensions/epub/metadata.schema.json

File: streamer/parser/metadata.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ The goal of this document is to provide directions that each implementation of R
 
 While the default context is very flexible in the way each metadata can be represented, when parsing a publication in the streamer we always use the most complex form for each metadata to harmonize our output.
 
+Related Repository: [Readium’s Webpub Manifest](https://github.com/readium/webpub-manifest)
```
Comment (HadrienGardeur) [inline streamer/parser/metadata.md]: Might be better to use the full name here: Readium Web Publication Manifest

File: streamer/parser/metadata.md (hunk 1 above)
Comment (JayPanoz) [reply streamer/parser/metadata.md]: Done

File: streamer/parser/metadata.md [hunk 3]
```diff
@@ -291,45 +293,66 @@ Does not apply.
...
-| auto               | auto                |
-| paginated          | paginated           |
-| scrolled-doc       | scrolled            |
-| scrolled-continous | scrolled-continuous |
+| rendition:flow      | values                                        |
+|---------------------|-----------------------------------------------|
+| auto                | "overflow": "auto" + "continuous": false      |
+| paginated           | "overflow": "paginated" + "continuous": false |
+| scrolled-doc        | "overflow": "scrolled" + "continuous": false  |
+| scrolled-continuous | "overflow": "scrolled" + "continuous": true   |
```
Comment (JayPanoz) [inline streamer/parser/metadata.md]: @HadrienGardeur re `continuous: true` default change, isn’t the mapping table already covering it?

Comment (HadrienGardeur) [reply streamer/parser/metadata.md]: Yeah, seems so. I'll merge it now in this case.

**Pull request merged**